### PR TITLE
Listed missing HTTP and gRPC endpoint envvars used

### PR DESCRIPTION
**Pull request by WhitWaldo**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

I noticed that the [published page](https://docs.dapr.io/reference/environment/) listing the various environment variables was missing two of the values used today: `DAPR_HTTP_ENDPOINT` and `DAPR_GRPC_ENDPOINT`, so I've added them to the list.

## Issue reference

N/A

### CODE REVIEW

**Review by hhunter-ms — APPROVED** [review]
lgtm

File: daprdocs/content/en/reference/environment/_index.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ The following table lists the environment variables used by the Dapr runtime, CL
 | APP_PORT             | Dapr sidecar | The port your application is listening on  |
 | APP_API_TOKEN        | Your application | The token used by the application to authenticate requests from Dapr API. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information. |
 | DAPR_HTTP_PORT       | Your application | The HTTP port that the Dapr sidecar is listening on. Your application should use this variable to connect to Dapr sidecar instead of hardcoding the port value. Set by the Dapr CLI run command for self-hosted or injected by the `dapr-sidecar-injector` into all the containers in the pod.                                   |
+| DAPR_HTTP_ENDPOINT   | Your application | The HTTP endpoint that the Dapr sidecar is listening on. Your application should use this variable to change the endpoint used to connect to the Dapr sidecar instead of hardcoding an alternative hostname. It is expected that this value minimally include the schema and hostname.|
```
Comment (msfussell) [inline daprdocs/content/en/reference/environment/_index.md]: Suggest providing an example here to make this clear "minimally include the schema and hostname"

File: daprdocs/content/en/reference/environment/_index.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ The following table lists the environment variables used by the Dapr runtime, CL
...
 | DAPR_GRPC_PORT       | Your application | The gRPC port that the Dapr sidecar is listening on. Your application should use this variable to connect to Dapr sidecar instead of hardcoding the port value. Set by the Dapr CLI run command for self-hosted or injected by the `dapr-sidecar-injector` into all the containers in the pod.                                   |
+| DAPR_GRPC_ENDPOINT   | Your application | The gRPC endpoint that the Dapr sidecar is listening on. Your application should use this variable to change the endpoint used to connect to the Dapr sidecar instead of hardcoding an alternative hostname. It is expected that this value minimally include the schema and hostname.|
```
Comment (msfussell) [inline daprdocs/content/en/reference/environment/_index.md]: Same here. Suggest providing an example here to make this clear "minimally include the schema and hostname"

**Review by msfussell — COMMENTED** [review]
@WhitWaldo - These are great missing additions. See my comments on examples to help developers.

File: daprdocs/content/en/reference/environment/_index.md (hunk 2 above)
Comment (WhitWaldo) [reply daprdocs/content/en/reference/environment/_index.md]: Added an example for HTTP to differentiate from the above example.

File: daprdocs/content/en/reference/environment/_index.md (hunk 1 above)
Comment (WhitWaldo) [reply daprdocs/content/en/reference/environment/_index.md]: Added an example indicating HTTPS.

**Comment by WhitWaldo** [comment]
@msfussell For both examples, I'll come back and edit the language in the docs if/when another [PR](https://github.com/dapr/dotnet-sdk/pull/1339) I created today is accepted and merged.

**Pull request merged**